docs/design/arc-onboarding.md

```diff
@@ -0,0 +1,27 @@
+# Azure Arc for Kubernetes integration
+
+## Motivation
+
+Simplify Arc agents deploymet by making it an optional last step of the cluster deployment process.
+
+## High level implementation
+
+- First, add `connectedClusterProfile` to apimodel
+
+```json
+{
+    "connectedClusterProfile": {
+      "tenantID": "azure-spn-tenant",
+      "subscriptionID": "azure-sub",
+      "resourceGroup": "connected-cluster-rg-azure",
+      "clusterName": "friendly-name",
+      "clientID": "azure-spn",
+      "clientSecret": "azure-spn-secret",
+      "region": "azure-region"
+    }
+}
+```
+
+- AKSe replaces the placeholders in [arc-onboarding.yml](https://github.com/Azure/azure-arc-kubernetes-onboarding/blob/master/arc-onboarding.yml), base64 encodes the yaml and save the onboarding manifest in the control plane nodes.
```

Comment:
Let's always refer to AKS Engine as "AKS Engine"

Reply:
This file will go away now that I am actually working on the feature